### DISCUSSION TITLE
woof/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#40] [Announcements]
# woof/

It Turns Out

https://www.supergoodcode.com/woof/

**Comment by qwertychouskie** [comment]
"current record-holder for Most Tests Fixed With One Line Of Code"

I need to hear this story

**Reply by zmike** [reply]
https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/14932 close enough

**Reply by qwertychouskie** [reply]
But that MR was created by you, not... unless... you two are the same person?!?

**Reply by zmike** [reply]
Look at the commits

**Reply by qwertychouskie** [reply]
Ah, so you have 3 identities then, got it ;)
